data/events/2020-medellin.yml

```
@@ -14,8 +14,8 @@ startdate: 2020-07-30T08:00:00-06:00 # The start date of your event. Leave blank
 enddate: 2020-07-31T08:00:00-06:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
-cfp_date_start:  # start accepting talk proposals.
-cfp_date_end:  # close your call for proposals.
+cfp_date_start:  2020-01-23T08:00:00-06:00 # start accepting talk proposals.
+cfp_date_end:  2020-04-18T08:00:00-06:00 # close your call for proposals.
 cfp_date_announce:  # inform proposers of status
 
 cfp_link: "" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
```

You have an open Call For Papers, but you haven't got the link listed here. You'll get higher visibility if you add the link. 

```suggestion
cfp_link: "https://www.papercall.io/devopsdaysmedellin"
```